modules/openapi-generator/src/test/java/org/openapitools/codegen/protobuf/ProtobufSchemaCodegenTest.java

```diff
@@ -55,399 +54,433 @@ public void testCodeGenWithAllOf() throws IOException {
         Map<String, String> globalProperties = new HashMap<>();
         // set line break to \n across all platforms
         System.setProperty("line.separator", "\n");
-
         File output = Files.createTempDirectory("test").toFile();
-        List<File> files = generate(output, properties, globalProperties, "src/test/resources/3_0/allOf_composition_discriminator.yaml");
-        TestUtils.ensureContainsFile(files, output, "models/pet.proto");
-        Path path = Paths.get(output + "/models/pet.proto");
-        assertFileEquals(path, Paths.get("src/test/resources/3_0/protobuf-schema/pet.proto"));
-
-        output.delete();
+        try {
```

Comment:
why a try/finally ?

Reply:
To guarantee the deletion of the temporary files even in cases of an exception / error